Makefile.am

```diff
@@ -1355,10 +1355,10 @@ imap_pop3d_LDADD = $(LD_SIEVE_ADD) $(LD_SERVER_ADD)
 PROMDATAGEN = $(abs_top_srcdir)/imap/promdatagen
 
 imap/promdata.c: imap/promdata.p $(PROMDATAGEN)
-	$(AM_V_GEN)$(PROMDATAGEN) -c $@.NEW $< && mv $@.NEW $@
+	$(PERL) $(AM_V_GEN)$(PROMDATAGEN) -c $@.NEW $< && mv $@.NEW $@
```

Comment:
Strange -- then why is there no space between the `$(AM_V_GEN)` and script name?

BTW, the build won't succeed even with the `#!` fixes _unless_ it is guaranteed that the script files retain their execute bits.

Reply:
Because `$(AM_V_GEN)` is nothing to do with perl, it's automake magic.

The command being run here is `$(PROMDATAGEN) ...` (yes, relying on +x and a valid #! line).  To change the command to run the `$(PERL)` executable with `$(PROMDATAGEN)` as an argument, the `$(PERL)` needs to be part of the command, and thus after the automake magic -- otherwise, make ends up passing the expanded automake magic as an argument to `($PERL)` (which complains), and then accidentally running `$(PROMDATAGEN)` directly anyway (again: relying on +x and a valid #! line), at least in my case, because on my system the expanded automake magic includes a `;`...